bin/benchmark_pydy_code_gen.py

```diff
@@ -9,15 +9,18 @@
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 from pydy.models import n_link_pendulum_on_cart
-from sympy import symbols
 
+from pydy.backend import sm, USE_SYMENGINE
 
 def run_benchmark(max_num_links, num_time_steps=1000):
     """Runs the n link pendulum derivation, code generation, and integration
     for each n up to the max number provided and generates a plot of the
     results."""
 
-    methods = ['lambdify', 'theano', 'cython']
+    if USE_SYMENGINE:
+        methods = ['lambdify']
+    else:
+        methods = ['lambdify', 'theano', 'cython']
```

Comment:
because symengine doesn't have code generation capabilities.
Note that `symengine+lambdify` is as fast as `sympy+cython` at runtime and has much less initial overhead.